gen3/bin/dcf.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+
+source "${GEN3_HOME}/gen3/lib/utils.sh"
+gen3_load "gen3/gen3setup"
+
+AWS_INPUT_BUCKET="data-refresh-manifest"
+AWS_OUTPUT_BUCKET="data-refresh-output"
+GS_INPUT_BUCKET="replication-input"
+GS_OUTPUT_BUCKET="datarefresh-log"
+
+echo "Hello from dcf!"
+command=$1
+release=$2
+
+# Activate google service account
+if [ -f "${WORKSPACE}/${vpc_name}/apis_configs/dcf_dataservice/creds.json" ]; then
+  gcloud auth activate-service-account --key-file=${WORKSPACE}/${vpc_name}/apis_configs/dcf_dataservice/creds.json
+  export GOOGLE_APPLICATION_CREDENTIALS=${WORKSPACE}/${vpc_name}/apis_configs/dcf_dataservice/creds.json
+fi
+
+generate_aws_refresh_report() {
+  manifest_type=$3
+  manifest="$(aws s3 ls s3://$AWS_INPUT_BUCKET | grep GDC_full_sync_${manifest_type}_.*$release | awk -F' ' '{print $4}')"
+
+  if [ -z "$manifest" ]; then
+    echo "fail to download $manifest_type manifest!!!"
+    exit 1
+  fi
+
+  refresh_log=${manifest//.tsv/.txt}
+
+  echo "$refresh_log"
+
+  if [ ! -f "/tmp/$manifest" ]; then
```

Comment:
Use `mktemp` to setup temporary files, and avoid using `/tmp/` - $XDG_RUNTIME_DIR points to a temp directory that is particular to the logged in user - ex:
```
manifestLocal="$(mktemp $XDG_RUNTIME_DIR/manifest_XXXXXX)"
```

Reply:
I cache the file here so that we don't have to re-download the manifest files again. Thanks!